fhir-server/src/main/java/org/linuxforhealth/fhir/server/filter/rest/FHIRRestServletFilter.java

```diff
@@ -135,7 +138,11 @@ public void doFilter(HttpServletRequest request, HttpServletResponse response, F
             FHIRConfiguration.validateTenantId(tenantId);
```

Comment:
Now that we're checking the tenantId against a preconfigured list of tenants, maybe we don't need to validate it here any more?

Reply:
This validation is for allowed characters for a tenant Id. Can this be removed?

Reply:
I think it could be removed if we want, but it shouldn't hurt to keep it and could potentially provide a better error message than the generic "no config for this tenant".